packages/core/src/sass/app/modules/welcome.scss

```diff
@@ -1,5 +1,6 @@
 .dc-modal__container_welcome {
     overflow: visible;
+    background-color: $color-grey-12;
```

Comment:
please use color scheme from `theme` if it is new color, make sure to add new variable in theme that will have both dark and light theme

Reply:
yes, it is new. waiting for a design decision. I will update whenever i get an answer